CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 5.1.0 (in progress)
 ================
-* Your contribution here
+* [#1221](https://github.com/oshi/oshi/pull/1221): Added messages to unit test assertions in GraphicsCardTest.java. - [@david145noone](https://github.com/david145noone).
```

Comment:
As you may have noticed, this PR ended up with the number 1222, so you'll have to edit these two numbers.

Reply:
Thanks for the advice and the feedback, I've incorporated the changes you suggested!